lib/sidekiq/client.rb

```diff
@@ -218,6 +219,7 @@ def normalize_item(item)
       item['queue'.freeze] = item['queue'.freeze].to_s
       item['jid'.freeze] ||= SecureRandom.hex(12)
       item['enqueued_at'.freeze] ||= Time.now.to_f
```

Comment:
Should this property be set inside `atomic_push` instead when we know the job is actually being enqueued?